README.md

```diff
@@ -26,13 +26,28 @@ from a scikit-learn background.
 
 ## Installation
 
+### pip
+
 Pmdarima has binary and source distributions for Windows, Mac and Linux (`manylinux`) on pypi
 under the package name `pmdarima` and can be downloaded via `pip`:
 
 ```bash
-$ pip install pmdarima
+pip install pmdarima
```

Comment:
Removed the dollar sign because GitHub has a new feature where you can just copy the whole code block. So if someone wants to use that to copy/paste, it would cause issues.

![Screen Shot 2021-08-03 at 11 07 45 AM](https://user-images.githubusercontent.com/21350310/128049154-3a37c3c6-2881-4cea-a74d-db60e55427ee.png)